src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/EnvironmentVariablesConfigurationProvider.cs

```diff
@@ -15,6 +15,7 @@ public class EnvironmentVariablesConfigurationProvider : ConfigurationProvider
         private const string MySqlServerPrefix = "MYSQLCONNSTR_";
         private const string SqlAzureServerPrefix = "SQLAZURECONNSTR_";
         private const string SqlServerPrefix = "SQLCONNSTR_";
+        private const string PostrgeSqlPrefix = "POSTGRESQLCONNSTR_"
```

Comment:
I think you should use this in location around this
https://github.com/dotnet/runtime/blob/3d97f41f084953ff7dc85776b82f976e6a22919e/src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/EnvironmentVariablesConfigurationProvider.cs#L67-L70

Reply:
Also you should add test for this here https://github.com/dotnet/runtime/blob/3d97f41f084953ff7dc85776b82f976e6a22919e/src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/tests/EnvironmentVariablesTest.cs#L64-L65

Reply:
My bad. Adding the missing part